python/lsst/pipe/supertask/activator.py

```diff
@@ -131,6 +132,34 @@ def activate(self):
         """
         pass
 
+def get_inheritance(stask_class, subclasses=[]):
```

Comment:
If you were never using it without providing the second argument it might have been safer to make it a mandatory argument rather than optional.